cluster/gce/gci/configure-helper.sh

```diff
@@ -752,10 +752,10 @@ function start-kube-apiserver {
   fi
   local -r src_dir="${KUBE_HOME}/kube-manifests/kubernetes/gci-trusty"
 
-  if [[ -n "${KUBE_USER:-}" ]]; then
+  if [[ -n "${KUBE_USER}" ]]; then
```

Comment:
This should still be `"${KUBE_USER:-}"` if this script is supposed to work without KUBE_USER being set.